core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -522,20 +507,36 @@ export class ContributionsAndReview
   ngOnInit(): void {
     this.SUGGESTION_TYPE_QUESTION = 'add_question';
     this.SUGGESTION_TYPE_TRANSLATE = 'translate_content';
+    this.ACCOMPLISHMENTS_TYPE_STATS = 'stats';
+    this.ACCOMPLISHMENTS_TYPE_BADGE = 'badges';
     this.TAB_TYPE_CONTRIBUTIONS = 'contributions';
     this.TAB_TYPE_REVIEWS = 'reviews';
     this.TAB_TYPE_ACCOMPLISHMENTS = 'accomplishments';
+    this.TAB_TYPES_TO_NAMES = {
```

Comment:
This is a constant, so just populate it directly when declared? Not sure why you have it empty at the top and overwrite it here later.